packages/hs-react-ui/src/components/Card/Card.tsx

```diff
@@ -144,6 +144,10 @@ const Card = ({
   disableFeedback,
   feedbackType = FeedbackTypes.ripple,
 }: CardProps): JSX.Element | null => {
+  const hasContent = Boolean(children);
```

Comment:
would either call this `hasBody` or `hasChildren` to be specific to the nomenclature here 😄

Reply:
Good call.

Reply:
I will name it hasBody